rewriting/task1/quick_sort.c

```diff
@@ -0,0 +1,35 @@
+#include "quick_sort.h"
+
+int distribution(int arr[], int low, int high)
```

Comment:
У этой функции есть стандартное название:
```suggestion
int partition(int arr[], int low, int high)
```